model/security/patch_action_groups/structures.smithy

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+@input
+structure PatchActionGroups_Input{
+    content: PatchActionGroupsParams
```

Comment:
Patch action group takes a list directly so this should be of type `PatchOperationList`. It also needs the `@httpPayload` trait added